webpack.config.dev.js

```diff
@@ -18,6 +18,11 @@ module.exports = {
                 query: {
                     presets: ['es2015']
                 }
+            },
+            {
+              test: /\.css$/,
+              exclude: /node_modules/,
+              loader: "style-loader!css-loader"
```

Comment:
Could you please put an inline comment documenting this line? I sort of think I know what style-loader does, but I read all the docs and could not figure out why you need the `!css-loader` part, or what it does. Also  note you should be using single quotes.